config/puma.rb

```diff
@@ -1,28 +1,30 @@
+# frozen_string_literal: true
```

Comment:
I think you could avoid edit these(config, etc...) rails files and only take a look of `"your files"`

Reply:
Ok ok we will put it in exceptions, I think we already made it, but we will review again.